tests/common/bakery_chain/baker.rs

```diff
@@ -81,4 +81,22 @@ impl Related<super::cake::Entity> for Entity {
     }
 }
 
+pub struct BakedForCustomer;
+
+impl Linked for BakedForCustomer {
+    type FromEntity = Entity;
+
+    type ToEntity = super::customer::Entity;
+
+    fn link() -> Vec<RelationDef> {
```

Comment:
I think we should have self here, i.e. `fn link(&self)`

Reply:
Okay, will update it now.

Btw... what is the reason behind this? lolll

Reply:
I think it will open a lot of possibilities when the struct is not a unit struct.

Reply:
Got it